### PR TITLE
Added text commands

### PR DESCRIPTION
**Pull request by larsenv**
FFS why is it showing still old commits

### CODE REVIEW

**Comment by Seriell** [comment]
Let me test it actually works before merging this time :^)

**Comment by awesomebing1** [comment]
It shows your old commits because I just copied it directly. Seriel I advise you do the same

**Comment by Seriell** [comment]
You advise I do what?

**Comment by awesomebing1** [comment]
Copy the JSON directly and commit instead of squashing and merging

**Comment by Seriell** [comment]
Is there a specific reason for that?
Does merging mess something up?

**Comment by awesomebing1** [comment]
From my understanding, it would tuck away the fun translation commands into history, which isn't what I want. Up to you however.

**Comment by Seriell** [comment]
The new text commands obviously work fine, but the translation ones don't seem to work and Oreo is broken(?)
image
Not sure what's going on here, someone else should take a look.

File: modules/commands.rb [hunk 1]
```diff
@@ -46,6 +46,8 @@ class << self
             support: "⚙ **Need help?**\nYou can join the support server here:\n**https://discord.gg/43SaDy6 **",
             facedesk: 'https://giphy.com/gifs/XLOsdacfjL5cI',
             smea: 'https://giphy.com/gifs/Sb2NkTl1oV6eI',
+            milk: 'I am a frequent ice chewer and I recently discovered "milk" :wink: ice. Firstly you put "milk" :wink: in the ice tray and then you let it freeze. I got so addicted to this that I covered my whole freezer with "milk" :wink: ! Of course nothing beats the original water so sometimes I shave a half cup of water with half cup of "milk" :wink: and crunch away. It' + "'" + 's probably bad for me but I also try "lemon" :wink: mixed with "milk" :wink: :wink:',
```
Comment (awesomebing1) [inline modules/commands.rb]: Why isn't this in quotes, with escaped quotes as needed?

File: modules/commands.rb [hunk 2]
```diff
@@ -46,6 +46,8 @@ class << self
...
+            mine: "This mailbox :mailbox: is mine :thumbsup: and this triangular sign :zzz: that blue balloon :balloon: the month of June :calendar: there mine mine mine mine Mine ziggy's sweets :candy: are mine :thumbsup: the birdies tweets are mine! :thumbsup: city streets :walking: both ur feet :feet: there all emphatically mine :thumbsup: it's all belong to me :open_hands: everything that I see :see_no_evil: north south east and west I caress it :open_hands: cause I possess it, I'm stingy and it's mine!!!!! :thumbsup::thumbsup::thumbsup: snap snap snap :ok_hand::ok_hand: And this instrumental break is also mine :thumbsup::thumbsup: the floor :thumbsdown: and ceiling :thumbsup: are mine all your feeling are mine :slight_smile::confused::cry::angry::thinking:  you always knew it! That's all there is to it! It's mine mine mine mine mine :raised_hands: that's what I said it's mine!",
```
Comment (awesomebing1) [inline modules/commands.rb]: Shouldn't we change this to the actual emojis?

Comment (awesomebing1) [inline modules/commands.rb]: Actually, we shouldn't include these in the first place. They're too meme-y for this bot.

**Comment by awesomebing1** [comment]
We decided not to add the meme-y commands after all. I fixed the Oreo issue manually.

Also, @larsenv, please delete your current fork and resync. Your commit history is out of sync, causing the extra comments here.

**Pull request closed without merging**